test/Microsoft.AspNetCore.Mvc.DataAnnotations.Test/Internal/DataAnnotationsMetadataProviderTest.cs

```diff
@@ -159,12 +169,54 @@ public void CreateBindingMetadata_EditableAttributeTrue_SetsReadOnlyFalse()
             Assert.False(context.BindingMetadata.IsReadOnly);
         }
 
+        [Fact]
+        public void CreateDisplayMetadata_DisplayAttribute_NameFromResources_UsesDataAnnotationLocalizaterProvider()
+        {
+            // Arrange
+            var sharedLocalizer = new Mock<IStringLocalizer>(MockBehavior.Loose);
+            sharedLocalizer
+                .Setup(s => s["DisplayName"])
+                .Returns(() => null);
```

Comment:
I didn't really care what the result from the `IStringLocalizer` was, I've gone ahead and made what I'm looking for more explicit and less round-about on the next pass.